packages/editor-ui/src/components/NodeDetailsView.vue

```diff
@@ -210,16 +209,16 @@ export default defineComponent({
 		const pinnedData = usePinnedData(activeNode);
 		const router = useRouter();
 		const workflowHelpers = useWorkflowHelpers({ router });
+		const workflowActivate = useWorkflowActivate();
 
 		return {
 			externalHooks,
 			nodeHelpers,
 			pinnedData,
 			workflowHelpers,
+			workflowActivate,
```

Comment:
Is this actually used in this component? I cannot see any references to it in the code or template 🤔

Reply:
Also, I think we can remove the arguments from the `setup` function (`props` & `ctx`) since they are not used.

Reply:
workflow active there is calling `activateCurrentWorkflow` method